dashboard/app/models/programming_expression.rb

```diff
@@ -208,7 +208,13 @@ def summarize_for_show
   end
 
   def summarize_for_lesson_show
-    {name: name, color: color, syntax: syntax, link: documentation_path}
+    {
+      name: name,
+      blockName: block_name,
+      color: color,
+      syntax: syntax,
+      link: documentation_path
```

Comment:
is it more semantically correct to call this `link` on the frontend? or should we call this property `documentationPath` to match the backend?

Reply:
oh good call. I'm just changing spacing here so I'll fix this as a follow-up